Readme.md

```diff
@@ -182,7 +185,41 @@ const NotFoundPage = loadable(() => import("src/pages/NotFoundPage"), {
 });
 ```
 
-`loadable-components` were deprecated in favour of `@loadable/component`, but `@loadable/component` dropped `getState`. So if you want to use `loadable-components` you can use old version (`2.2.3` latest version at the moment of writing) or you can wait until `React` will implement proper handling of this case with asynchronous rendering and `React.lazy`.
+> `loadable-components` were deprecated in favour of `@loadable/component`, but `@loadable/component` dropped `getState`. So if you want to use `loadable-components` you can use old version (`2.2.3` latest version at the moment of writing) or you can wait until `React` will implement proper handling of this case with asynchronous rendering and `React.lazy`.
+
+2. Use [react-prerendered-component](https://github.com/theKashey/react-prerendered-component)
```

Comment:
this is next level dark magic 😍. Looking forward to try it out.

Reply:
At your beck and call 🤓